include/dlaf/eigensolver/bt_band_to_tridiag.h

```diff
@@ -7,9 +7,10 @@
 // Please, refer to the LICENSE file in the root directory.
 // SPDX-License-Identifier: BSD-3-Clause
 //
+
 #pragma once
 
-#include "dlaf/eigensolver/bt_band_to_tridiag/mc.h"
+#include "dlaf/eigensolver/bt_band_to_tridiag/impl.h"
```

Comment:
Good catch!!!

I will fix it here, while about #491 I will fix it in #493 or in a separate PR (I will try checking if these are the only mistakes or there are more) 😉

Reply:
I've just opened #498 for starting the discussion about this. Since we don't have a decision yet, we can decide to revert https://github.com/eth-cscs/DLA-Future/pull/496/commits/615fd900e1da548cb5421cf92b2cc7ef88d2a2ca or not.